content/applications/marketing/email_marketing.rst

```diff
@@ -1,440 +1,792 @@
+- :guilabel:`Mailing Contact`: focuses on specific mailing contacts in the database.
+- :guilabel:`Sales Order`: focuses on a specific sales orders in the database.
 
-If the specified recipient fields don't automatically reveal themselves, simply click the
-:guilabel:`Add Filter` button beneath the :guilabel:`Recipients` field, and Odoo reveals the
-necessary equation fields to further drill down the target recipients for this mailing.
+Add recipient filter
+~~~~~~~~~~~~~~~~~~~~
 
-Add a recipient filter
-----------------------
+To add a more specific recipient filter to any :guilabel:`Recipient` option, select any recipient
+option (other than :guilabel:`Mailing List`), and click the :guilabel:`Modify filter (right-facing
+arrow)` icon beneath the :guilabel:`Recipient` field to reveal three subsequent filter rule fields,
+formatted like an equation.
 
-To add a more specified recipient filter, select any recipient option (other than :guilabel:`Mailing
-List`), and clicking :guilabel:`Add Filter`, if needed, to reveal three fields, formatted like an
-equation.
+To reveal the sub-menu options within the filter rule fields, click each field, and make the desired
+selections, until the preferred configuration has been achieved.
 
-To reveal the sub-menu options, click each field, and make the desired selections, until the
-preferred configuration has been achieved. The number of :guilabel:`Records` that match the rule(s)
-are indicated to the right of the :guilabel:`Recipients` field, in green.
+The number of :guilabel:`records` in the database that match the configured rule(s) are indicated
+beneath the configured filter rule(s), in green.
 
-.. image:: email_marketing/add-filter-button.png
+.. image:: email_marketing/filter-records.png
    :align: center
    :alt: View of how recipient filters can be customized in Odoo Email Marketing.
 
 .. note::
    Some sub-menu options in the first rule field allow for a second choice to provide even more
    specificity.
 
-To the right of each rule, are :guilabel:`× (Delete node)`, :guilabel:`＋ (Add node)`, and
-:guilabel:`⋯ (Add branch)` icons.
+To the right of each rule, there are :guilabel:`➕ (Add node)`, :guilabel:`(Add branch)`, and
+:guilabel:`🗑️ (Delete node)` icons.
 
-The :guilabel:`× (Delete node)` icon deletes a specific node (line) of the rule. The :guilabel:`＋
-(Add node)` icon adds a node (line) to the rule. And, the :guilabel:`⋯ (Add branch)` icon adds a
-branch to the node. A branch means two additional, indented sub-nodes are added to the rule,
-providing even more specificity to the line above it.
+The :guilabel:`➕ (Add node)` icon adds a node (line) to the rule.
 
-Mail body tab
--------------
+The :guilabel:`(Add branch)` icon adds a branch to the node. A branch means two additional, indented
+sub-nodes are added to the rule, providing even more specificity to the line above it.
+
+The :guilabel:`🗑️ (Delete node)` icon deletes a specific node (line) of the rule.
```

Comment:
Would turn this into a bullet list 👍 
The emoji labels are incorrect, as well; what we write in parenthesis is a fail-safe label in case the emoji fails to load. So `➕ (Add node)` is actually `➕ (plus sign)`. I touched up the definitions slightly, for your consideration.

```suggestion
To the right of each rule, there are three additional options, represented by :guilabel:`➕ (plus
sign)`, :guilabel:`(sitemap)`, and :guilabel:`🗑️ (trash)` icons.

- The :guilabel:`➕ (plus sign)` icon adds a new node (line) to the overall targeting logic.
- The :guilabel:`(sitemap)` icon adds a branch to the node. A branch contains two additional,
  indented sub-nodes that are related to that specific rule, providing even more specificity to the
  parent line above it.
- The :guilabel:`🗑️ (trash)` icon deletes a specific node (line) in the array of logic
```